watertap/tools/oli_api/flash.py

```diff
@@ -573,7 +573,13 @@ def configure_flash_analysis(
                     }
                 )
 
-        input_dict["inflows"] = inflows
+        if ("unit" in inflows.keys()) and ("values" in inflows.keys()):
+            input_dict["inflows"] = inflows
+        # otherwise, assume a solute dictionary with {solute_name: concentration_value}
+        else:
+            unit = self.input_unit_set["molecularConcentration"]["oli_unit"]
+            inflows_oli_names = {get_oli_name(k): v for k, v in inflows.items()}
+            input_dict["inflows"] = {"unit": unit, "values": inflows_oli_names}
```

Comment:
Another important change to allow for the user to just supply a solute:concentration dict to the `configure_flash_analysis` method for setting up isothermal flash calculations. 

This is what we do for our `configure_water_analysis` method, and this `configure_flash_analysis` seems to assume it'll be getting output from `get_apparent_species_from_true`; i.e., without this PR, the code assumes that the user is running a water analysis first and then extracting apparent species output as input to this method; either that, or the user needs to manually supply a dict that doesn't match the input dict format that goes into `configure_water_analysis`. 

This standardizes things to make things less confusing for the user.